packages/fluentui/react-northstar/src/components/Accordion/Accordion.tsx

```diff
@@ -93,6 +93,9 @@ export interface AccordionProps extends UIComponentProps, ChildrenComponentProps
    * Accessibility behavior if overridden by the user.
    */
   accessibility?: Accessibility<AccordionBehaviorProps>;
+
+  /** Manage if panels should be rerendered each time or just shown or hidden. */
+  disablePanelsRerendering?: boolean;
```

Comment:
```suggestion

  /** Manage if panels should be rendered always or based on their state. */
  alwaysRenderPanels?: boolean;
```

Please rename the prop as it's not about rerendering, it's what me initially confused 🙃